colander/tests/test_colander.py

```diff
@@ -2787,6 +2787,32 @@ def test_clone(self):
         self.assertEqual(inner_clone.name, 'inner')
         self.assertEqual(inner_clone.foo, 2)
 
+    def test_clone_with_modified_schema_instance(self):
```

Comment:
+1, It's helpful with regressions like this if the failing test is in a commit prior to the fix because it makes it trivial to confirm the issue.